### PR TITLE
[docs] Fix titles and links 

### PR DESCRIPTION
**Pull request by aishwarya24**

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Back up and restore YugabyteDB Anywhere
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:2]: the titles are duplicates - headertitles don't matter/are fine...

File: docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ aliases:
 type: docs
 ---
 
-YugabyteDB Anywhere allows you to use its UI or [API](https://api-docs.yugabyte.com/docs/yugabyte-platform/) to manage asynchronous replication between independent YugabyteDB clusters. You can perform deployment via unidirectional (master-follower) or [bidirectional](#set-up-bidirectional-replication) (multi-master) xCluster replication between two data centers.
+YugabyteDB Anywhere allows you to use its UI or [API](https://api-docs.yugabyte.com/docs/yugabyte-platform/f10502c9c9623-yugabyte-db-anywhere-api-overview) to manage asynchronous replication between independent YugabyteDB clusters. You can perform deployment via unidirectional (master-follower) or [bidirectional](#set-up-bidirectional-replication) (multi-master) xCluster replication between two data centers.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md:15]:
```suggestion
YugabyteDB Anywhere allows you to use its UI or [API](../../anywhere-automation/anywhere-api/) to manage asynchronous replication between independent YugabyteDB clusters. You can perform deployment via unidirectional (master-follower) or [bidirectional](#set-up-bidirectional-replication) (multi-master) xCluster replication between two data centers.
```

File: docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ aliases:
...
 
 In the concept of replication, universes are divided into the following categories:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md:18]: xcluster links below are absolute

File: docs/content/preview/develop/learn/data-types-ycql.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Data types in YCQL
```
Comment (ddhodge) [inline docs/content/preview/develop/learn/data-types-ycql.md:2]: isn't the problem the title tag is duplicated

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md (hunk 1 above)
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:2]:
```suggestion
title: Back up and restore YugabyteDB Anywhere using yb-ctl
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
-headerTitle: Back up and restore YugabyteDB Anywhere
+headerTitle: Back up and restore YugabyteDB Anywhere on YBA Installer
 description: Use a script to back up and restore YugabyteDB Anywhere on YBA Installer.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:4]:
```suggestion
description: Back up and restore YugabyteDB Anywhere using YBA Installer.
```

File: docs/content/preview/develop/learn/data-types-ycql.md (hunk 4 above)
Comment (aishwarya24) [reply docs/content/preview/develop/learn/data-types-ycql.md:2]: Right, sorry i got confused.

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Back up and restore YugabyteDB Anywhere
-headerTitle: Back up and restore YugabyteDB Anywhere
-description: Use a script to back up and restore YugabyteDB Anywhere on YBA Installer.
+title: Back up and restore YugabyteDB Anywhere using yb-ctl
+headerTitle: Back up and restore YugabyteDB Anywhere on YBA Installer
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-installer.md:3]:
```suggestion
headerTitle: Back up and restore YugabyteDB Anywhere using YBA Installer
```

File: docs/content/preview/explore/ycql-language/data-types.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Data types in YCQL
+title: Data types
```
Comment (ddhodge) [inline docs/content/preview/explore/ycql-language/data-types.md:2]:
```suggestion
title: YCQL Data types
```

Comment (ddhodge) [reply docs/content/preview/explore/ycql-language/data-types.md:2]: unless that conflict with something else

File: docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 8]
```diff
@@ -22,8 +22,8 @@ In the concept of replication, universes are divided into the following categori
 
 For additional information on xCluster replication in YugabyteDB, see the following:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md:24]: Both links broken

File: docs/content/stable/yugabyte-platform/create-deployments/async-replication-platform.md [hunk 9]
```diff
@@ -21,8 +21,8 @@ In the concept of replication, universes are divided into the following categori
 
 For additional information on xCluster replication in YugabyteDB, see the following:
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/create-deployments/async-replication-platform.md:23]: same

**Review by ddhodge — APPROVED** [review]

**Pull request merged**